.github/workflows/python-test-coverage.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Test Coverage
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        echo -e "import nltk\nnltk.download('punkt')" | python3
+        echo -e "import nltk\nnltk.download('averaged_perceptron_tagger')" | python3
+        echo -e "import nltk\nnltk.download('stopwords')" | python3
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        coverage run
+        coverage xml
+    - name: Upload Coverage Results
+      uses: codecov/codecov-action@v1
+      with:
+        file: coverage.xml
+        fail_cli_if_error: true
```

Comment:
Coverage is throwing a warning, I believe because of this line -- I think this parameter is called `fail_ci_if_error` (note ***ci*** not ***cli***)

Reply:
Ah! I completely missed that. Thank you!